src/lib.rs

```diff
@@ -150,14 +157,7 @@ pub fn bytes_of<T: Pod>(t: &T) -> &[u8] {
 /// empty slice might not match the pointer value of the input reference.
 #[inline]
 pub fn bytes_of_mut<T: Pod>(t: &mut T) -> &mut [u8] {
```

Comment:
If you replace `T: Pod` with `T: NoPadding + AnyBitPattern`, then any `Pod` type would be valid, but also other types that (at least in generic context) are known to implement `NoPadding` and `AnyBitPattern`

Reply:
I guess I'm in the wrong timezone to participate in discord discussions.

My point is that `T: Pod` is unnecessary strict compared to `T: NoPadding + AnyBitPattern`.
I can imagine some generic context where generic type has `NoPadding + AnyBitPattern` bound.
If that code is generated by proc-macro it may be hard to replace with `Pod`.
I don't have real case of this naturally. But we can play it safe.

Reply:
Sorry for the delay, but I can agree with this. Changed in d5abea1